core/domain/question_services.py

```diff
@@ -674,15 +721,11 @@ def save_question_summary(question_summary):
         question_summary: QuestionSummaryModel. The question summary object to
             be saved in the datastore.
     """
-    question_summary_model = question_models.QuestionSummaryModel(
-        id=question_summary.id,
-        question_model_last_updated=question_summary.last_updated,
-        question_model_created_on=question_summary.created_on,
-        question_content=question_summary.question_content,
-        misconception_ids=question_summary.misconception_ids,
-        interaction_id=question_summary.interaction_id
+    existing_question_summary_model = (
+        question_models.QuestionSummaryModel.get(question_summary.id))
+    question_summary_model = populate_question_summary_model_fields(
+        existing_question_summary_model, question_summary
     )
```

Comment:
Can we move this closing paren to above line?

Reply:
I think yes, but I usually prefer this style (except for some cases).

Reply:
Okay np from my side, and everything else looks good.
Thank you.